src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,10 @@
+@mixin square($size) {
+  width: $size;
+  height: $size;
+  border-radius: $border-radius;
+}
+
+@mixin border($radius) {
```

Comment:
The mixin name 'border' is not descriptive enough. Consider renaming it to 'border-top-radius' or something more specific to indicate its purpose.